src/shogun2-core/src/test/resources/META-INF/spring/mail-test-context.xml

```diff
@@ -11,7 +11,7 @@
 
     <context:property-placeholder location="classpath*:META-INF/*.properties" />
 
-    <context:component-scan base-package="de.terrestris.shogun2.util" />
+    <context:component-scan base-package="de.terrestris.shogun2.util.mail" />
```

Comment:
No, but the mail test failed after the commit from before, so this fix should be included in this PR, shouldn't it?